### ISSUE TITLE
documentation update: descriptor and resource details

### ISSUE THREAD

**Issue by gwbischof** [bluesky/event-model#78]
I think the documentation for descriptors and resources should be improved:
http://nsls-ii.github.io/event-model/data-model.html#resource-document
http://nsls-ii.github.io/event-model/data-model.html#event-descriptor

It isn't clear how to understand the meaning of the different parts of these documents.

**Comment by gwbischof** [comment]
https://nsls-ii.github.io/bluesky/event_descriptors.html#

Here is some documentation already created for descriptors

**Comment by danielballan** [comment]
I agree that the documentation should be improved. Action item here should be to port the narrative text from https://blueskyproject.io/bluesky/documents.html and https://nsls-ii.github.io/architecture-overview.html and https://nsls-ii.github.io/bluesky/event_descriptors.html into this documentation and then delete those so that we can maintain it all in one place.

**Comment by jklynch** [comment]
I'm looking at https://blueskyproject.io/bluesky/documents.html. Should this file go in to the event-model documentation as-is, or should it be combined with an existing file? Also it references the callbacks page in the bluesky documentation. Should it continue to reference that?

**Comment by danielballan** [comment]
My sense of it at this point is:

- https://blueskyproject.io/bluesky/documents.html should be combined into https://blueskyproject.io/event-model/data-model.html. The "Overview of a Run" section can just be dropped, I think, but the bulleted lists that describe the contents of each document type are worth merging into the cooresponding sections in data-model.html. Any links to sections of the bluesky documentation should probably be dropped.
- The descriptors page https://nsls-ii.github.io/bluesky/event_descriptors.html should be moved wholesale to event-model and adapted if needed to remove bluesky links.
- This paragraph from the architecture-overview.html should be merged in as well:

> The scope of a “run” is left to the beamline or individual scientist. It is quite generic: a set of Documents generated by following a given sequence of instructions. This might encompass a single short measurement (say, “a count”) or a multi-step procedure such as a raster scan. A run should represent a set of individual measurements that will be collected or processed together.
